tools/check-swagger

```diff
@@ -0,0 +1,22 @@
+var SwaggerParser = require('swagger-parser');
+
+function validateSwagger(file) {
+    SwaggerParser.validate('static/yaml/zulip.yaml')
+        .then(function() {
+            // The validation passed, no need to say anything
+            return;
+        })
+        .catch(function(err) {
+            // There is something wrong. Display the validation errors
+            console.error('In', file + ':');
+            console.error(err.message);
+            process.exitCode = 1;
+        });
+}
+
+// Iterate through the changed files, passed in the arguments.
+// The two first arguments are the call to the Node interpreter and this
+// script, hence the starting point at 2.
+for(var i = 2; i < process.argv.length; i++) {
+    validateSwagger(process.argv[i]);
+}
```

Comment:
So, if there are other YAML files in the repository, this will end up validating the zulip.yaml file once for every YAML file in the repository, which is buggy in a couple ways.

I think you probably want to do a few things:
* In a new commit at the start of this branch, move `static/yaml` to `static/swagger`, so we have a clear directory where things that are specifically swagger files (as opposed to just YAML files) go.
* Have this loop check if the argument starts with `static/swagger`, and if it doesn't, ignore it.
* Have validateSwagger not ignore the file argument and just validate the file passed in.